### PR TITLE
L1 OF example

### PR DESCRIPTION
**Pull request by szalpal**
Signed-off-by: Michał Szołucha <mszolucha@nvidia.com>

### CODE REVIEW

File: docs/examples/optical_flow/optical_flow_example.ipynb [hunk 1]
```diff
@@ -0,0 +1,593 @@
+{
+<<<<<<< Updated upstream
```
Comment (Kh4L) [inline docs/examples/optical_flow/optical_flow_example.ipynb]: please remove these rebase artifacts

**Comment by Kh4L** [comment]
What exactly the relation with #705 ?

File: dali/pipeline/operators/optical_flow/optical_flow.cc [hunk 2]
```diff
@@ -12,8 +12,8 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-#include "dali/pipeline/data/views.h"
-#include "dali/pipeline/operators/optical_flow/optical_flow.h"
+#include <dali/pipeline/data/views.h>
```
Comment (jantonguirao) [inline dali/pipeline/operators/optical_flow/optical_flow.cc]:
```suggestion
#include "dali/pipeline/data/views.h"
```

File: Dockerfile [hunk 3]
```diff
@@ -48,7 +48,7 @@ ENV CC=${CC}
 ENV CXX=${CXX}
 
 RUN LD_LIBRARY_PATH="${PWD}:${LD_LIBRARY_PATH}" && \
-    cmake ../ -DCMAKE_INSTALL_PREFIX=. \
+    cmake ../ -DCMAKE_INSTALL_PREFIX=. -DCMAKE_BUILD_TYPE=Debug \
```
Comment (klecki) [inline Dockerfile]: Plz no. This breaks the build.

File: qa/L1_jupyter_notebooks_TU102/test.sh [hunk 4]
```diff
@@ -3,7 +3,7 @@
 source ../setup_test.sh
 
 # used pip packages
-pip_packages="jupyter"
+pip_packages="jupyter matplotlib mxnet-cu##CUDA_VERSION## tensorflow-gpu torchvision torch"
```
Comment (JanuszL) [inline qa/L1_jupyter_notebooks_TU102/test.sh]: Do you use any of "mxnet, tensorflow, torch" in the example?

File: qa/L1_jupyter_plugins/test.sh [hunk 5]
```diff
@@ -17,8 +17,9 @@ mkdir -p idx_files
 cd docs/examples
 
 test_body() {
-    # dummy patern
-    black_list_files="#"
+    black_list_files="optical_flow_example.ipynb\|#" # optical flow requires TU102 architecture
+                                                     # whilst currently L1_jupyter_plugins test
+                                                     # can be run only on V100
```
Comment (JanuszL) [inline qa/L1_jupyter_plugins/test.sh]: I would rather say that 'L1_jupyter_plugins can be run on any configuration, including non TU102'

File: docs/examples/optical_flow/optical_flow_example.ipynb [hunk 6]
```diff
@@ -22,16 +22,16 @@
    "outputs": [
     {
      "ename": "ImportError",
-     "evalue": "libdali.so: cannot open shared object file: No such file or directory",
+     "evalue": "/home/mszolucha/virtualenv/dali/lib/python3.5/site-packages/nvidia/dali/backend_impl.so: undefined symbol: _Py_ZeroStruct",
```
Comment (jantonguirao) [inline docs/examples/optical_flow/optical_flow_example.ipynb]: I guess this should not be here

**Pull request merged**